proposals/lip-0037.md

```diff
@@ -43,7 +43,7 @@ Note that any updates to the serialization specification of a message type must
 Chain identifiers for transaction signatures and block signatures were already introduced in [LIP 0009](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0009.md#specification) and [LIP 0024](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0024.md#update-to-the-block-header-signing-procedure) (as network identifiers) to prevent replay attacks on other chains. Here, we specify how to use a chain identifier for any kind of message to prevent replay attacks on other chains in general.
 These specifications supersede the specifications given in [LIP 0009][lip-0009].
 
-Chain identifiers are 4-byte values that follow a specific format: the first byte is used to identify the network in which the chain is running (either the mainnet, official testnet, or any other test network); the other 3 bytes identify the chain within the network. We include the network-specific prefix explicitly to ensure that a chain does not use the same chain identifier in the test network as in the mainnet.
+Chain identifiers are 4-byte values that follow a specific format: the first byte is used to identify the network in which the chain is running (either the mainnet, official testnet, or any other test network). The first bit of the first byte is set to `0`. This is done to prevent the overflow of the max size of `uint32` when we hardened the paths for [Ed25519 derivation](https://github.com/LiskHQ/lips/blob/main/proposals/lip-0066.md#ed25519-key-derivation). The other 3 bytes identify the chain within the network. We include the network-specific prefix explicitly to ensure that a chain does not use the same chain identifier in the test network as in the mainnet.
```

Comment:
If we want to add the explanation, I would do so in another paragraph.

Reply:
Done.